test/swar/BasicOperations.cpp

```diff
@@ -0,0 +1,19 @@
+#include "zoo/swar/metaLog.h"
+#include "zoo/swar/SWAR.h"
+
+#include "catch2/catch.hpp"
+
+#include <type_traits>
+
+using namespace zoo;
+
+TEST_CASE(
+    "Basic SWAR",
+    "[swar]"
+) {
+    uint8_t v = 0x3;
+    SECTION("Simple Mask") {
+      auto mask = swar::makeBitmask<8, uint8_t>(v);
+      CHECK(mask == 0x08080808ull);
```

Comment:
In the interest of brevity, perhaps we should make a 32 bit or perhaps a 16 bit test instead, of nibbles?